ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -48,6 +48,7 @@ public override async Task<IActionResult> OnGetAsync(int? id, bool isBeingSubmit
                 return NotFound();
             }
 
+            NotificationBannerModel = new NotificationBannerModel(Notification);
```

Comment:
Might it make more sense to initialise this in the base method OnGetAsync (which is currently abstract but doesn't need to be) in NotificationBase? Obviously we'd have to add a call to base.OnGetAsync into each of these models then but it would mean not having to reference NotificationBannerModel in the derived classes at all

Reply:
Forgot to reply to this! I think @jasiekmiko and I talked roughly about this, I'm happy to go with either way so happy to make the change if wanted!

Reply:
@ChristianMcc I've refactored the notificationbase model quite a bit in my current ticket, so let's wait until this is all merged to make any other changes